snuba/web/rpc/common/exceptions.py

```diff
@@ -29,3 +29,10 @@ def convert_rpc_exception_to_proto(
         inferred_status = 429
 
     return ErrorProto(code=inferred_status, message=str(exc))
+
+
+def is_400(status_code: int) -> bool:
```

Comment:
I don't know it needs its own function if we just check the range like I suggested, or maybe call it `is_status_code_4xx`?